spacy/cli/templates/quickstart_training.jinja

```diff
@@ -32,9 +32,10 @@ batch_size = {{ 128 if hardware == "gpu" else 1000 }}
 factory = "transformer"
 
 [components.transformer.model]
-@architectures = "spacy-transformers.TransformerModel.v1"
+@architectures = "spacy-transformers.TransformerModel.v3"
 name = "{{ transformer["name"] }}"
 tokenizer_config = {"use_fast": true}
+transformer_config = {"output_attentions": false}
```

Comment:
I would not add this to the quickstart. It is empty by default (it will be filled in) and that is fine.

Reply:
Ah, sure. I don't have a strong opinion on that.